CWE/decisions/bulgaria.txt

```diff
@@ -38,4 +38,36 @@ political_decisions = {
 		
 	}
 
+
+	bul_decom_namechange = {
+		picture = "transition_to_democracy"
+		potential = {
+			tag = BUL
+			NOT = {
+				OR = {
+					has_global_flag = bul_decom_namechange
+					has_country_flag = bulgarian_de_stalinization
+				}
+			}
+			NOT = { has_global_flag = 1992_start_date_flag }
```

Comment:
All decisions should have at least 1 positive check. Otherwise this decision can be fired immediately on day 1 of the 1946 or 1950 start.